src/components/common/ErrorsDisplay.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { array } from 'prop-types';
+
+const DisplayErrors = ({ errors }) => {
+  if (!errors) {
```

Comment:
Normally we want the control over a presence of a component in the parent's logic. 

So instead of controlling here if you have errors, you can have in the parent:
`errors && <DisplayErrors errors={errors} />`

Reply:
Thanks!